[Bug?]: stripeWebhook.test.js fails out of the box with Cannot find module '../../lib/stripe'

After setting up the example store on a new JS project, via the CLI's setup command, I went to try the test and it failed with:

```
    Cannot find module '../../lib/stripe' from 'api/src/functions/stripeWebhook/stripeWebhook.test.js'

      1 | import { mockHttpEvent } from '@redwoodjs/testing/api'
      2 |
    > 3 | import { stripe } from 'src/lib/stripe'
        | ^
      4 |
      5 | import { handler } from './stripeWebhook'
      6 |
```

The `api/src/lib/stripe` file is indeed missing; not sure if we forgot to scaffold it out or scoped it out of the design and the test needs updating

--- a/api/src/index.js
+++ b/api/src/index.js
@@ -1,5 +1,6 @@
 export {
     handleStripeWebhooks,
+    generateStripeMockEvent,
     stripe
 } from './lib'
 

--- a/api/src/lib/index.js
+++ b/api/src/lib/index.js
@@ -1,3 +1,4 @@
+import { mockHttpEvent } from '@redwoodjs/testing/api'
 
 import Stripe from 'stripe'
 
@@ -19,7 +20,7 @@ export const lastEntry = (array) => {
   return latest[latest.length-1]
 }
 
-export const handleStripeWebhooks = (event, context, webhooksObj, secure = true) => {
+export const handleStripeWebhooks = (event, context, webhooksObj = {}, secure = true) => {
   if (secure) {
     const endpointSecret = process.env.STRIPE_WEBHOOK_KEY
     try {
@@ -31,7 +32,8 @@ export const handleStripeWebhooks = (event, context, webhooksObj, secure = true)
       )
 
       // Find event type in webhookObject
-      if (typeof webhooksObj[stripeEvent.type] !== 'undefined') {  
+      if (typeof webhooksObj[stripeEvent.type] !== 'undefined') {
+        // execute function for the event
         webhooksObj[stripeEvent.type](stripeEvent, context)
       }
 
@@ -57,4 +59,36 @@ export const handleStripeWebhooks = (event, context, webhooksObj, secure = true)
       throw error
     }
   }
+}
+
+// Generates a test Stripe mock event
+export const generateStripeMockEvent = () => {
+
+const payload = JSON.stringify(
+    {
+      id: 'evt_test_webhook',
+      object: 'event',
+    },
+    null,
+    2
+  )
+
+  process.env.STRIPE_WEBHOOK_SK = 'whsec_test_secret'
+
+  /**
+   * @see {@link https://github.com/stripe/stripe-node/blob/master/README.md#testing-webhook-signing}
+   */
+  const header = stripe.webhooks.generateTestHeaderString({
+    payload,
+    secret: process.env.STRIPE_WEBHOOK_SK,
+  })
+
+  const httpEvent = mockHttpEvent({
+    body: payload,
+    headers: {
+      'stripe-signature': header,
+    },
+  })
+
+  return httpEvent
 }

--- a/cli/templates/api/src/functions/stripeWebhook/stripeWebhook.test.js
+++ b/cli/templates/api/src/functions/stripeWebhook/stripeWebhook.test.js
@@ -1,41 +1,13 @@
-import { mockHttpEvent } from '@redwoodjs/testing/api'
-
-import { stripe } from 'src/lib/stripe'
-
-import { handler } from './stripeWebhook'
+import {
+  handleStripeWebhooks,
+  generateStripeMockEvent,
+} from '@redwoodjs-stripe/api'
 
 describe('stripeWebhooks function', () => {
   it('Should respond with 200', async () => {
-    /**
-     * Copied from Stripe's tests: {@link https://github.com/stripe/stripe-node/blob/master/test/Webhook.spec.js#L8-L12}
-     */
-    const payload = JSON.stringify(
-      {
-        id: 'evt_test_webhook',
-        object: 'event',
-      },
-      null,
-      2
-    )
+    const mockStripeEvent = generateStripeMockEvent()
 
-    process.env.STRIPE_WEBHOOK_SK = 'whsec_test_secret'
-
-    /**
-     * @see {@link https://github.com/stripe/stripe-node/blob/master/README.md#testing-webhook-signing}
-     */
-    const header = stripe.webhooks.generateTestHeaderString({
-      payload,
-      secret: process.env.STRIPE_WEBHOOK_SK,
-    })
-
-    const httpEvent = mockHttpEvent({
-      body: payload,
-      headers: {
-        'stripe-signature': header,
-      },
-    })
-
-    const response = await handler(httpEvent, null)
+    const response = await handleStripeWebhooks(mockStripeEvent, {}, {}, false)
 
     expect(response.statusCode).toBe(200)
   })

